src/client/styles.css

```diff
@@ -753,4 +753,38 @@ body {
     border: none;
     border-radius: 2rem;
     cursor: pointer;
+}
+
+/* Style for SignInView */
+
+/* Styles specifically for the email and password input fields */
+.text-input input[type='email'],
+.text-input input[type='password'] {
```

Comment:
you can jut do `text-input input {}` and it'll apply to `input[type='email'` and `input[type='password'`

Reply:
Also, to make it specific to SignInView, apply SignInView's class: `sign-in-container .text-input input {}`. You do have to do this, otherwise it'll mess with all the other views since they all share the same css file.